### PR TITLE
Feature/vtn 20075 fix openalpr cannot create assets causing by engine toolkit

### PR DESCRIPTION
**Pull request by seta-davenguyen**
When upgrading openalpr engine to use the latest code of engine-toolkit. I encountered this bug https://steel-ventures.atlassian.net/projects/VTN/issues/VTN-20075. 

I tried at https://github.com/veritone/task-openalpr-enginetoolkit-chunk-go/blob/feature/VTN_20075_Fix_Openalpr_can_not_create_assets/main.go#L162 with [json.NewEncoder(w).Encode(resp) or w.Write(res)] but both did not work. edge_output_writer reported json encode error when creating vtnStandard.

### CODE REVIEW

File: engine/engine.go [hunk 1]
```diff
@@ -247,8 +292,12 @@ func (e *Engine) processMessageMediaChunk(ctx context.Context, msg *sarama.Consu
 		ChunkUUID:     mediaChunk.ChunkUUID,
 		StartOffsetMS: mediaChunk.StartOffsetMS,
 		EndOffsetMS:   mediaChunk.EndOffsetMS,
```
Comment (matryer) [inline engine/engine.go]: Is it `endOffsetMS` or `stopOffsetMS`?

File: engine/engine.go [hunk 2]
```diff
@@ -223,8 +257,14 @@ func (e *Engine) processMessageMediaChunk(ctx context.Context, msg *sarama.Consu
 		if resp.StatusCode != http.StatusOK {
 			return errors.Errorf("%d: %s", resp.StatusCode, buf.String())
 		}
+		e.logDebug("Toolkit got output from openalpr", buf.String())
 		if buf.Len() > 0 {
-			content = buf.Bytes()
+			if err := json.NewDecoder(&buf).Decode(&engineOutputContent); err != nil {
```
Comment (matryer) [inline engine/engine.go]: Can we tweak the structure here to be more like:

```go
if buf.Len() == 0 {
  ignoreChunk = true
  return nil
}
if err := json.NewDecoder(... // etc
```

File: engine/engine.go (hunk 1 above)
Comment (seta-davenguyen) [reply engine/engine.go]: It is endOffsetMs in json.

File: engine/engine.go (hunk 2 above)
Comment (seta-davenguyen) [reply engine/engine.go]: Yeah, this is nicer. I updated

File: engine/engine.go (hunk 1 above)
Comment (matryer) [reply engine/engine.go]: Is that what it should be according to VTN standard?

Comment (seta-davenguyen) [reply engine/engine.go]: Can you explain more clearly? Do you mean it should be stopOffsetMS? With this it worked. Assets created.

**Pull request merged**